CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ Latest – to be added to the next release
 ------------------------------
 ### Added
 - `CODEOWNERS` file with the `@justeat/ui-senior-reviewers` set as reviewers for `pie-microsite`
+- Root level devDependency of `@justeattakeaway/pie-icons`
```

Comment:
You could just bump the version in the CHANGELOG here – it's only an arbitrary version so we can track the changes going into the monorepo